module/x/gravity/keeper/keeper.go

```diff
@@ -152,28 +152,59 @@ func (k Keeper) iterateEthereumSignatures(ctx sdk.Context, storeIndex []byte, cb
 //  ORC -> VAL ADDRESS //
 /////////////////////////
 
```

Comment:
These changes only include cleaning the legibility of existing logic and adding new `Delete*` APIs.